### DISCUSSION TITLE
High resolution custom milestones for print

### DISCUSSION THREAD

**Discussion by M4CC** [bafolts/terraforming-mars#4442] [General]
Hello,

Can you tell me where I can find the high resolution images for the custom milestones?
I wanted to use them with my physical copy of the game but it seems you compressed the images quite a lot for the web version so I'm wondering if you'd be able to point me to the original images you used.

Thanks in advance!

**Comment by alrusdi** [comment]
We use separate repository to store the art sources https://github.com/terraforming-mars/terraforming-mars-assets-sources
But i don't sure if millestone images can be found there

**Reply by M4CC** [reply]
@alrusdi thank you for your reply. I had already checked and couldn't find it there.

**Comment by M4CC** [comment]
@ssimeonoff perhaps you know? I see you edited the milestone images and removed the text. Thanks in advance!
